### PR TITLE
Alchemy Logo

### PR DESCRIPTION
**Pull request by LaNeraNura**
** Checklist **
- [x] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x] I am confident that all icons are clear and easy to read/understand
- [x] I have provided a link to the brand icon’s brand guidelines whenever possible.

If this pull request is not adding new icons, you can remove this checklist.

### CODE REVIEW

File: app/assets/images/payment_icons/alchemy.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...AAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAA AACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUA AAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAA AAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo 0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAA AAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAA AACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEA AAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAAAIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAA AAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAAAAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBK NAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAAAAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAA AAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAAIgSTQEAAAAAAAAAAAAAAAAAokRTAAAAAAAAAAAAAAAA AIAo0RQAAAAAAAAAAAAAAAAAIEo0BQAAAAAAAAAAAAAAAACIEk0BAAAAAAAAAAAAAAAAAKJEUwAA AAAAAAAAAAAAAACAKNEUAAAAAAAAAAAAAAAAACBKNAUAAAAAAAAAAAAAAAAAiBJNAQAAAAAAAAAA AAAAAACiRFMAAAAAAAAAAAAAAAAAgCjRFAAAAAAAAAAAAAAAAAAgSjQFAAAAAAAAAAAAAAAAACja drVi+4q57eTdAAAAAElFTkSuQmCC" id="image883" clip-path="url(#clipPath889)" transform="matrix(.02667 0 0 .02667 49.861 13.826)" opacity="1" stop-opacity="1"/></svg>
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/alchemy.svg:1]: There a few elements that I don't see used.

- `<path id="path836">`
- `<path id="rect866">`
- `<clipPath id="clipPath864">`
- `<path id="rect891">`

Can we also remove `id="svg840"`?

In summary, can we please remove anything that is not used?

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi,

I left some comments.

Please let me know if there are any questions.

**Comment by LaNeraNura** [comment]
Hi Thomas, I have re-generated the icon. Kindly check again.

**Review by tauthomas01 — CHANGES_REQUESTED** [review]
Hi, I still see some `defs` and `id` that are unused. Can you please check and remove those attributes if there are unused?
Thank you

**Comment by LaNeraNura** [comment]
Okay, I have deleted the parameters.

**Comment by tauthomas01** [comment]
Hi @LaNeraNura ,

I still see some `id` instances that are not used. Can we make sure that all unused `id` attributes are removed?

Also, I noticed I can't view the SVG file (it's freezing when I view on Chrome). It seems like the file is oddly heavy (466kb) which is really big for a SVG file.

Can you check if we can remove things that are hidden or not used?

File: app/assets/images/payment_icons/alchemy.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...XXL6KhUAAAAAACAHaOent/tzp2bJFu3Dg0l SX9/rex4+OHh4brXvHl1/6lx2bJaALJ0aZ2vWpUkw8MLF7Zt0uutXFlKMjhYC0SSPfao45w59c6z dVZraH7ef1Gt6piOhSzMBLXwqJTdd6+VL1M3gPjnpv6d1Vc8si1J+vp+/OM6//a32zYZHr777lrI c+utSTIwcNddvV4yMvKtbyXJxEQd2/ZHP6rfT/X1fX3/9E91ex1/8pNaYDMwUJ+fnHzgAX9nAAAA APCL1cLoqVW817ymbZMNGy65JEmuueaHP5QQAAA8fopmAABgO1u8+MAD6wmrZ5xRT2hdsEAqAAAA AADAttTtThXC7LprkpSy225J0ustWlTnUwUwu+1WT81ftixJmmbRorZN2nb58iTpdJYsqZ9n1Ne3 bR1L6e//Rf/fymGYPWrhUinz5z/23/XatVOPauHSz34PPPhgkvT13X57La353vfqM3fdVUoyMDA1 //a3e71kYuK7363zO+6o36f1wpi2rfs1zV13Jck999x5Z5LceONDD/m7AQAAAGBnU8rUStzGjUnS 6Zx9dp2/+9113LpVSgAA8KspmgEAgO1kzZq5c5smaZqzzqpbjj5aKgAAAAAAwBOxceOSJcmji2D2 3TdJmmblyloYs8cedc8VK2phzMKFdV6LZZpmwYLHFuHvsksd+/qSWpjxy4pilMjA4zE4WMf99qvf M/vt94u+f5omaduHHqqFNN//ft32ox/VZ+++u36//+AHpSQjIz/4QZJ0u7fd1uslyf/9v3W///N/ kqTX+8d/rK//1reSZHLy3nv9XQAAAAAwm9Q1trquXco557Rt0u1efXWSTE5+5SsSAgCAX03RDAAA bCe77LLvvm2blPK859UF7oEBqQAAAAAAAEkyOjo8nCRDQwcdlCT9/cceW4tj9tuvFlAcemiStO3u u9dXzJmTJKXMnVvnU+PQUB0HBn5VaQwwPZTS31+/V5ctq1umxqnnH7t/295/f932wAN1Sy2U6XTu u2/q+STpdr/xjfr55Je+VLd//et1/89+Nkluv/0b30iSW2998EF/CwAAAADMHJ1OkpRy7LFJ0ran nJIkq1ZZ7wIAgMdD0QwAAGwHpSRN89rXlpK0bb2TqBO7AQAAAABg57FhwwEHJElf3zHH1AKZ8fFa IDM6WvdYt
```
Comment (tauthomas01) [inline app/assets/images/payment_icons/alchemy.svg:1]: Also, I see `pi-fps` and FPS as the title. Should we rename to `pi-alchemy` and Alchemy as title?

**Comment by rahilvirani** [comment]
Closing the PR because it has been stale for a while. If the change is still relevant, please re-create the pull request using the new pull request template. We are sorry for the inconvenience.

**Pull request closed without merging**